infra.lvn_tools/extend_boot/README.md

```diff
@@ -0,0 +1,40 @@
+# ansible-bash-extend-boot
```

Comment:
Please break the long lines into single sentence lines. Markdown will put them all together into the same paragraph as long as there is no empty line